src/firefly/js/visualize/CsysConverter.js

```diff
@@ -295,8 +327,17 @@ export class CysConverter {
      * returns {ImagePt}
      */
     static makeIPtFromIWPt(iwPt) {
-        if (!iwPt) return null;
-        return makeImagePt(iwPt.x, iwPt.y);
+        return iwPt ? makeImagePt(iwPt.x, iwPt.y) : null;
+    }
+
+    static makeIPtFromFitsImPt(pt) {
+        return pt ? makeImagePt(pt.x-.5, pt.y-.5) : null;
+    }
+
+    makeIPtFromxZeroImPt(pt) {
```

Comment:
could this be static?

Reply:
no, it references `this.projection.header`